applications/Defibox_vault.md

```diff
@@ -0,0 +1,234 @@
+# EOS Network Foundation Grant Proposal
+
+- **Project Name:** Vault Protocol
+- **Team Name:** Defibox
+- **EOS Payment Address:** liquid.defi
+- **[Level](https://github.com/eosnetworkfoundation/grant-framework#grant-levels):** 3
+- **Pomelo Grant(s):** https://pomelo.io/grants/vault
+- **Project is Open-Source:** Yes*
+- **Project was part of Token sale:** No
+- **Repository where Project resides:** https://github.com/DefiboxTeam/DefiboxVault-interface
+> Please only answer Yes or No to the Open-Source or Token sale questions.  Do not explain more on that line.  You may delete this line and all lines that begin with `>` from your application.
+
+## Contact
+
+- **Contact Name:** Raven
+- **Contact Email:** raven@defibox.io
+- **Website:** https://eos.defibox.io/vault
+
+> ⚠️ *The combination of your GitHub account submitting the application and the EOS account address above will be your unique identifier during the program. Please keep them safe.*
+
+
+## Open source - Timelines
+
+**Stage One (completed) "Version Beta v0.1.0"**
+
+> https://github.com/defiboxswap/DefiboxVault-interface
+
+- [x] Includes *.hpp interface
+- [x] Prebuilt WASM binaries *.wasm & *.abi (with version control)
+- [x] General documentation README.md
+- [x] Protocol Design (diagram)
+
+**Stage Two (not completed)**
+
+- [ ] Protocol Design Definition (translated in English/Korean/Chinese)
+- [ ] SC development (production)
+- [ ] User documentation
+- [ ] Automatic Contracts Tests
+
+**Stage Three (re-evaluation 2nd Security Audit from Certik) “Version v1.0.0”**
+
+- [ ] All smart contract source code will be made available (*.cpp)
```

Comment:
@Mattijah to answer your question here: https://github.com/eosnetworkfoundation/grant-framework/pull/98/files/e69fe19c14ec730df8c1bdb4a62869d4b29681d2#r1107862746

Reply:
> For security reasons, the *.cpp is not made available in the first stage, as the vault contains an excess of 9M EOS.

The vault will still contain the same amount of EOS even after the 1st stage, so I'm not sure how valid this argument is nevertheless if this is something that will be done then alright. Thanks for clarifying.